Makefile

```diff
@@ -80,14 +81,19 @@ lint:
 	$(ISORT) --check-only --diff $(EXAMPLES_DIR) $(PACKAGE) $(SETUP_PY) $(TESTS_DIR)
 	check-manifest
 
-coverage:
+coverage-clean:
 	$(COVERAGE) erase
-	$(COVERAGE) run "--include=$(PACKAGE)/*.py,$(TESTS_DIR)/*.py" --branch $(SETUP_PY) test
```

Comment:
This is no longer supported, plus it was broken. `python setup.py test` failed when it encountered `raise unittest.SkipTest(..)`, I simply migrated the code to reuse the same way `make test` works currently

Reply:
Already addressed.